src/main/java/com/conveyal/datatools/manager/jobs/MergeFeedsJob.java

```diff
@@ -0,0 +1,584 @@
+                            if (mergeFeedsResult.remappedIds.containsKey(key)) {
+                                mergeFeedsResult.remappedReferences++;
+                                // If the value has been remapped update the value to write.
+                                valueToWrite = mergeFeedsResult.remappedIds.get(key);
+                            }
+                            // If the current foreign ref points to another record that has been skipped, skip this
+                            // record and add its primary key to the list of skipped IDs (so that other references can
+                            // be properly omitted).
+                            if (mergeFeedsResult.skippedIds.contains(key)) {
+                                String skippedKey = String.join(":", table.name, idScope, keyValue);
+                                if (orderField != null) {
+                                    skippedKey = String.join(":",
+                                                             skippedKey,
+                                                             csvReader.get(getFieldIndex(fieldsFoundInZip, orderField))
+                                    );
+                                }
+                                mergeFeedsResult.skippedIds.add(skippedKey);
+                                skipRecord = true;
+                                continue;
+                            }
+                        }
+                        if (this.mergeType.equals(REGIONAL) && isKeyField && !val.isEmpty()) {
+                            // For regional merge, if field is a GTFS identifier (e.g., route_id, stop_id, etc.),
+                            // add scoped prefix.
+                            valueToWrite = String.join(":", idScope, val);
+                        }
+
+                        rowValues[specFieldIndex] = valueToWrite;
+                    }
+                    if (table.name.equals("routes") && rowValues[0].contains("BART")) {
+                        LOG.info("Route record: {}", rowValues[0]);
+                    }
+                    // Do not write rows that are designated to be skipped.
+                    if (skipRecord && this.mergeType.equals(MTC)) {
+                        mergeFeedsResult.recordsSkipCount++;
+                        continue;
+                    }
+                    String newLine = String.join(",", rowValues);
+                    if (!rowStrings.add(newLine)) {
+                        // The line already exists in the output file, do not append it again. This prevents duplicate
+                        // entries for certain files that do not contain primary keys (e.g., fare_rules and transfers) and
+                        // do not otherwise have convenient ways to track uniqueness (like an order field).
+                        // FIXME: add ordinal field/compound keys for transfers (from/to_stop_id) and fare_rules (?).
```

Comment:
Can you create an issue for this FIXME?

Reply:
https://github.com/conveyal/gtfs-lib/issues/223